lib/puppet/provider/mount.rb

```diff
@@ -14,6 +14,7 @@ def mount
     if Facter.value(:kernel) == 'Darwin'
       args << '-o' << options if options && options != :absent
     end
+    resource[:name].gsub!('\040', ' ')
```

Comment:
See if there's a way to do this without calling gsub everytime I'm calling the mount/unmount command.